Server/server.js

```diff
@@ -1,6 +1,17 @@
 const express = require('express');
+const databaseHandler = require('../database/index.js')
 const app = express();
 
 app.use(express.static(__dirname + '/../public'));
 
+app.get('/games', (req, res) => {
+  databaseHandler.getFilteredData((err, data) => {
+    if(err){
+      return console.log('error retrieving data', err)
```

Comment:
Should handle error instead of just console.log // for example res.write(500).send or something like that.